resolver/src/main/java/io/netty/resolver/InetNameResolver.java

```diff
@@ -40,32 +37,15 @@
      */
     protected InetNameResolver(EventExecutor executor) {
         super(executor);
-        loopbackAddress = SocketUtils.loopbackAddress();
     }
 
     /**
+     * @deprecated Use {@link SocketUtils#loopbackAddress()}.
```

Comment:
@Scottmitch I think this is a bad advice as `SocketUtils` is in the internal package.